src/server.js

```diff
@@ -495,8 +495,9 @@ function getClientConfiguration() {
 function performAuthentication(data) {
 	const socket = this;
 	let client;
+	let token = null;
 
-	const finalInit = () => initializeClient(socket, client, data.token || null, data.lastMessage || -1);
+	const finalInit = () => initializeClient(socket, client, token, data.lastMessage || -1);
```

Comment:
Why are we not using data.token anymore?

Reply:
Because it passes in the hashed token now. And only if the session exists for provided token.

Reply:
But it's not, it's passing in `null` everytime here. What even is the point in passing in null?

Reply:
It's set lower in the code before this function is called. Kinda the same way `client` is assigned.

Reply:
Oh I  completely missed that this was a function, duh. Alright, yup.